### ISSUE TITLE
Flag aria-disabled="true" on elements that have a disabled attribute

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#348]
Spec: https://w3c.github.io/html-aria/#att-disabled
> Developers SHOULD NOT use aria-disabled="true" on any element which also has a disabled attribute.
> Developers MUST NOT use aria-disabled="false" on any element which also has a disabled attribute.

Steps:
- go to https://w3c.github.io/html-aria/tests/disabled-test.html
- run accessibility checker
- note that there is a violation for aria-disabled="false" (test 3) because it conflicts.
- should also give a warning (or a violation!) for aria-disabled="true" with disabled, even though there's no conflict, because developers shouldn't be doing that.

**Comment by MHoov** [comment]
As of June 1
- [x] general rule description (SME & Dev)
- [x] testcases written (Dev)
- [ ] rule developed  (Dev)
- [ ] message written (SME)
- [ ] help written (SME)

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/348?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by shunguoy** [comment]
close as fixed per #800

**Comment by eirckr** [comment]
Validated
<img width="1450" alt="Screen Shot 2022-05-03 at 11 45 07 AM" src="https://user-images.githubusercontent.com/62436670/166500469-84e9e1c4-b56c-459c-b578-b3abd20f92c9.png">
